tchannel/peer_heap.py

```diff
@@ -0,0 +1,61 @@
+from __future__ import absolute_import
+
+from .container import heap
+from .container.heap import HeapOperation
+
+
+class PeerHeap(HeapOperation):
+    def __init__(self):
+        self.peers = []
+
+    def size(self):
+        return len(self.peers)
+
+    def lt(self, i, j):
+        return self.peers[i].score < self.peers[j].score
```

Comment:
This is mostly about naming:

Low score is better? Wasn't it the opposite when we were doing scores before? High score was for more desirable peers. Maybe the attribute should be renamed? "load"? It's some measure of how much load we think that peer is under, so as a load balancer, we pick the one we think is under the least load?